docs/docs/compliance/contrib-compliance.md

```diff
@@ -0,0 +1,101 @@
+# Custom Compliance Checks
+
+Trivy supports several different compliance checks. The details on compliance scanning with Trivy are provided in the [compliance documentation](../../docs/compliance/compliance.md).
+All of the Compliance Checks currently available in Trivy can be found in the `trivy-policies/specs/compliance/` directory ([Link](https://github.com/aquasecurity/trivy-policies/tree/main/specs/compliance)).
```

Comment:
`trivy-policies` repo on github name has been changed to `trivy-checks`, change should apply to all references in doc